maintainers/maintainer-list.nix

```diff
@@ -16946,5 +16946,11 @@
     githubId = 32876;
     name = "Diego Zamboni";
   };
+  ethinx = {
+    email = "eth2net@gmail.com";
+    github = "ethinx";
+    githubId = 965612;
+    name = "York Wong";
+  };
```

Comment:
Please keep the list sorted, thanks!

Reply:
now it should be at the right position, thanks @wegank